Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/GeometricFillTool.java

```diff
@@ -67,6 +68,10 @@ public GeometricFillTool(Context context, ToolType toolType) {
 			baseShape = BaseShape.RECTANGLE;
 		}
 
+		if (shapeOutlineWidth == 0) {
```

Comment:
Why do you check for 0? (I mean why a specific value, should this be `<=0`? Since you are in the constructor, you could just set the `shapeOutlineWidth` to 25 there. Even better: don't hardcode magic numbers, use a "DEFAULT_SHAPE_OUTLINE_WIDTH" constant to define the value 25.